src/sentry/static/sentry/app/components/navTabs.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function NavTabs(props) {
+  let {children, underlined, ...tabProps} = props;
+  let className = 'nav nav-tabs' + (underlined ? ' border-bottom' : '');
+  return (
+    <ul className={className} {...tabProps}>
+      {children}
+    </ul>
+  );
```

Comment:
If you wanted you could shorten this up by not destructuring `children` and letting that prop get passed in, and it can become a self closing tag.